css/index.css

```diff
@@ -112,6 +112,117 @@ p {
     justify-content: center;
 }
 
```

Comment:
For the individual boxes, the parent child .box contains all the styling for the boxes.  You only need to override the color in each box.